arviz/stats/diagnostics.py

```diff
@@ -550,7 +550,8 @@ def ks_summary(pareto_tail_indices):
     """
     _numba_flag = Numba.numba_flag
     if _numba_flag:
-        kcounts = histogram(pareto_tail_indices)
+        bins = np.asarray([-np.Inf, 0.5, 0.7, 1, np.Inf])
```

Comment:
we *should* move `bins` to line 551, so that we don't repeat ourselves on line 556.